pyiron_contrib/atomistics/runner/job.py

```diff
@@ -85,98 +71,114 @@ class RunnerFit(GenericJob, HasStorage):
 
-        - Mode 3: predict the total energy and atomic forces for a structure.
+        - Mode 3:
+            - energy
+            - forces
 
-    Examples:
+    Examples
+    --------
     Starting a new calculation (Mode 1):
 
     ```python
-        from pyiron import Project
-        from job import RunnerFit
-        import ase.io.runner.runner as io
+    from pyiron import Project
+    from job import RunnerFit
+    from runnerase import read_runnerase
+
+    # Create an empty sample project and a new job.
+    pr = Project(path='example')
+    job = pr.create_job(RunnerJob, 'mode1')
 
-        # Create an empty sample project and a new job.
-        pr = Project(path='example')
-        job = pr.create_job(RunnerFit, 'mode1')
+    # Import RuNNer settings from RuNNer input.nn file using ASE's I/O routines.
+    options = read_runnerconfig('./')
 
-        # Import training dataset and RuNNer settings from RuNNer input.data
-        # and input.nn files using ASE's I/O routines.
-        structures, options = io.read_runnerase('./')
+    # Read input structures from a TrainingContainer that was saved before.
+    container = pr['H2O_MD']
 
-        # Attach the information to the job.
-        job.structures = structures
-        job.input.update(options)
+    # Attach the information to the job.
 
-        # Set the RuNNer Mode to 1.
-        job.input.runner_mode = 1
+    job.add_training_data = container
+    job.parameters.update(options)
 
-        job.run()
+    # Set the RuNNer Mode to 1.
+    job.input.runner_mode = 1
+
+    job.run()
+    ```
 
     Restarting Mode 1 and running Mode 2:
 
     ```python
-        job = Project('example')['mode1'].restart('mode2')
-        job.input.runner_mode = 2
-        job.run()
+    job = Project('example')['mode1'].restart('mode2')
+    job.parameters.runner_mode = 2
+    job.run()
     ```
 
     Restarting Mode 2 and running Mode 3:
 
     ```python
-        job = Project('runnertest')['mode2'].restart('mode3')
-        job.input.runner_mode = 3
-        job.run()
+    job = Project('runnertest')['mode2'].restart('mode3')
+    job.parameters.runner_mode = 3
+    job.run()
     ```
     """
 
-    __name__ = 'RuNNer'
+    __name__ = 'RuNNerJob'
+
     # These properties are needed by RuNNer as input data (depending on the
     # chosen RuNNer mode).
     _input_properties = ['scaling', 'weights', 'sfvalues', 'splittraintest']
 
-    def __init__(self, project, job_name, **kwargs):
+    # Define a default executable.
+    _executable = Executable(
+        codename='runner',
+        module='runner',
+        path_binary_codes=state.settings.resource_paths
+    )
+
+    def __init__(self, project: Project, job_name: str) -> None:
         """Initialize the class.
 
-        Args:
-            project (Project): The project container where the job is created.
-            job_name (str):  The label of the job (used for all directories).
+        Parameters
+        ----------
```

Comment:
```suggestion
        Args:
```

We generally go with google style in the rest of the code base.

Reply:
Sorry, I missed that in the pyiron_base `CONTRIBUTING.rst`. I will change that in a future commit.